airflow/providers/mysql/hooks/mysql.py

```diff
@@ -182,7 +184,7 @@ def bulk_dump(self, table, tmp_file):
         conn.commit()
 
     @staticmethod
-    def _serialize_cell(cell, conn):  # pylint: disable=signature-differs
+    def _serialize_cell(cell: object, conn: Connection) -> object:  # pylint: disable=signature-differs
```

Comment:
Maybe we can also match the signatures. I believe the "signature-differs" pylint exception can be removed by adding = None for conn parameter. Which is no harm at all.